cmd/kube-controller-manager/app/core.go

```diff
@@ -401,8 +401,7 @@ func startEphemeralVolumeController(ctx ControllerContext) (http.Handler, bool,
 		if err != nil {
 			return nil, true, fmt.Errorf("failed to start ephemeral volume controller: %v", err)
 		}
-		// TODO (before beta at the latest): make this configurable similar to the EndpointController
-		go ephemeralController.Run(1 /* int(ctx.ComponentConfig.EphemeralController.ConcurrentEphemeralVolumeSyncs) */, ctx.Stop)
+		go ephemeralController.Run(int(ctx.ComponentConfig.EphemeralController.ConcurrentEphemeralVolumeSyncs), ctx.Stop)
 		return nil, true, nil
```

Comment:
Thanks for taking care of this TODO. It never made it into tracking list of opens for the feature and then of course was promptly forgotten - my bad.

Making it configurable makes sense. The rest seems to be mostly cut-and-paste from how it is done elsewhere and looks good to me.

/lgtm